### PR TITLE
Add /issue and issues/ tests

### PR DESCRIPTION
**Pull request by katiemcculloch**

### CODE REVIEW

File: cypress/integration/utils/sharedTests/issuePage.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    newShouldArgs("be.visible.and.contain", ["Create issue", "Cancel"]),
+  ),
+];
+
+const newIssueButtonClickFlow = (loggedIn: boolean): ClickFlow =>
+  newClickFlow(
+    "[data-cy=new-issue-button]",
+    loggedIn
+      ? [...testLoggedInSignInTo, ...testNewIssueForm]
+      : [...testLoggedOutSignInTo("create an issue")],
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/issuePage.ts:123]: Can this just be `: testLoggedOutSignInTo("create an issue")`

File: cypress/integration/utils/sharedTests/signInTo.ts [hunk 2]
```diff
@@ -12,3 +13,16 @@ export const testSignInTo = (msg: string) => [
...
+    "should not suggest sign in for a logged in user",
+    "[data-cy=sign-in-to-do-something-prompt]",
+    newShouldArgs("not.exist"),
+  ),
+  newExpectation(
+    "should not show sign in link component for logged in user",
+    "[data-cy=sign-in-with-redirect-link]",
+    newShouldArgs("not.exist"),
+  ),
+] as Tests;
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/signInTo.ts:28]: What does this `as Tests` do?

Comment (tbantle22) [reply cypress/integration/utils/sharedTests/signInTo.ts:28]: If this needs a return type I think `export const testLoggedInSignInTo: Tests = [` is better practice

**Review by tbantle22 — APPROVED** [review]
LGTM

File: cypress/integration/utils/sharedTests/issuePage.ts (hunk 1 above)
Comment (katiemcculloch) [reply cypress/integration/utils/sharedTests/issuePage.ts]: Yes, thanks, fixed: https://github.com/dolthub/dolthub-cypress/pull/84/commits/37133008069578f475a1670958b719517f5220b6#diff-abc29f16aff1ed8358957a888ce0dbc4501f3b84d5524257f1b28579e77e6efaR123

**Pull request merged**